# Array of 3D Person skeleton in world frame
# pose_publisher/Skeleton3D17[] skeleton_3d_17
std_msgs/Float64MultiArray skeleton_3d_17_flat
uint16[] skeleton_3d_17_flat_shape
std_msgs/Float64MultiArray transform
uint16[] transform_shape